compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -186,6 +189,7 @@ class CompilationTests extends ParallelTesting {
 
   @Test def runAll: Unit = {
     compileFilesInDir("../tests/run", defaultOptions) +
+    compileFilesInDir("../tests/run", defaultOptimised) +
```

Comment:
If you run everything in `tests/run` with `defaultOptimised`, aren't the lines below redundant?

Reply:
Yes, I will remove them. Thanks